test/graphql/types/Organization/creator.test.ts

```
@@ -0,0 +1,469 @@
+          .map(() =>
+            resolveCreator(
+              mockOrganization,
+              {},
+              ctx as unknown as ResolverContext,
+            ),
+          );
+
+        const results = await Promise.all(requests);
+        results.forEach((result) => {
+          expect(result).toEqual(mockCreator);
+        });
```

Biome (1.9.4)

[error] Prefer for...of instead of forEach.

forEach may lead to performance issues when working with large arrays. When combined with functions like filter or map, this causes multiple iterations over the same type.

(lint/complexity/noForEach)